### PR TITLE
Panic fixed

### PR DESCRIPTION
**Pull request by furusiyya**

### CODE REVIEW

File: protocols/smb/smb.go [hunk 1]
```diff
@@ -38,6 +40,11 @@ type SMB struct {
 func ParseSMB(data []byte) (smb SMB, err error) {
 	smb = SMB{}
 	// HACK: Not sure what the data in front is supposed to be...
+	if !bytes.Contains(data, []byte("\xff")) {
```
Comment (glaslos) [inline protocols/smb/smb.go]: Can't you do a simple compare?

Comment (furusiyya) [inline protocols/smb/smb.go]: Compare on base of length?

Comment (glaslos) [inline protocols/smb/smb.go]: No, sorry, my bad. forgot how SMB works..

Comment (furusiyya) [inline protocols/smb/smb.go]: Its ok

**Pull request merged**